src/api/menu/MenuAPI.js

```diff
@@ -30,15 +30,15 @@ import Menu, { MENU_PLACEMENT } from './menu.js';
  * @property {Function} onDestroy callback function: invoked when menu is destroyed
  */
 
-/**
- * Popup Menu Item/action
- * @typedef {Object} Action
- * @property {String} cssClass Class for menu item
- * @property {Boolean} isDisabled adds disable class if true
- * @property {String} name Menu item text
- * @property {String} description Menu item description
- * @property {Function} callBack callback function: invoked when item is clicked
- */
+// /**
+//  * Popup Menu Item/action
+//  * @typedef {Object} Action
+//  * @property {String} cssClass Class for menu item
+//  * @property {Boolean} isDisabled adds disable class if true
+//  * @property {String} name Menu item text
+//  * @property {String} description Menu item description
+//  * @property {Function} callBack callback function: invoked when item is clicked
```

Comment:
As discussed `callback` is a bad name for this. Let's call it `onClick` so it's obvious how it gets triggered.